lightkurve/targetpixelfile.py

```diff
@@ -1126,12 +1126,24 @@ def _open_image(img, extension):
 
             if idx == 0:  # Get default keyword values from the first image
                 factory.keywords = hdu.header
+
+
+            # Get positional shift of the image compared to the reference WCS
+            wcs_current = WCS(hdu.header)
+            column_current, row_current = wcs_current.wcs_world2pix(
+                          np.asarray([[position.ra.deg], [position.dec.deg]]).T,0)[0]
```

Comment:
I'm worried this will break if `position` is equal to `None`.  A more robust solution may be to retrieve the ra/dec of the center pixel, and use that to compute the POS_CORRs?